src/main/java/net/minecraftforge/common/EnumPlantType.java

```diff
@@ -8,5 +10,14 @@
     Cave,
     Water,
     Nether,
-    Crop
-}
\ No newline at end of file
+    Crop;
+
+    public static EnumPlantType getPlantType(String name) 
+    {
+        for (EnumPlantType t : values()) 
```

Comment:
A comment in here noting that things should be standardized and a companion Documentation link would be good. 
Dont want to make it the norm to random plant types.